### ISSUE TITLE
Accessibility bug: VersionSwitcher does not have visible focus indicator

### ISSUE THREAD

**Issue by ashleemboyer** [chakra-ui/chakra-ui-docs#1667]
## 1. Environment

- Date encountered: Thursday, November 2nd, 2023
- Device: MacBook Air M1 2020
- OS: Ventura 13.5.1
- Browser: Google Chrome Version 117.0.5938.132

## 2. Steps to Reproduce

1. Go to [the home page](https://chakra-ui.com/)
2. Press `TAB` until the "Search the docs" `<button>` is focused
3. Press `TAB` one more time

## 3. Expected Result

The `<select>` rendered by `VersionSwitcher` **_should_** have a visible indicator.

## 4. Actual Result

The `<select>` rendered by `VersionSwitcher` **_does not_** have a visible focus indicator.

## 5. Screen recording

Here's a screen recording (with no sound) demonstrating the Actual Result I observed when running through the Steps to Reproduce.

https://github.com/chakra-ui/chakra-ui-docs/assets/43934258/53a65971-3c76-42fe-8b34-18bac04df2aa

## 6. Relevant Code Links

This `variant='unstyled'` property appears to be the line that is removing the visible focus indicator:

https://github.com/chakra-ui/chakra-ui-docs/blob/c0aa42001f0c79ce54e3c28797c7b7fb48573c75/src/components/version-switcher.tsx#L24

## 7. Relevant Resources

- [WCAG 2.1 Success Criterion 2.4.7 Focus Visible (Level AA)](https://www.w3.org/TR/WCAG21/#focus-visible): "Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible."

**Comment by ashleemboyer** [comment]
Just created a pull request that aims to fix this issue with 2 lines of code! 🤞🏻 

https://github.com/chakra-ui/chakra-ui-docs/pull/1668
